src/hooks/use_profile.ts

```diff
@@ -47,6 +57,19 @@ const fetchDesmosProfile = async (address: string) => {
   }
 };
 
+const fetchDesmosProfileDtag = async (dtag: string) => {
+  let data: DesmosProfileQuery = {
+    profile: [],
+  };
+  try {
+    data = await fetchDesmosDtag(dtag);
+    const formattedData = formatDesmosProfile(data);
+    return formattedData;
+  } catch (error) {
+    return null;
+  }
+};
+
```

Comment:
There seems to be a lot of duplications with `fetchDesmosProfile`. From the way I see it, you should be able to merge this with `fetchDesmosProfile`. If you prefer to have a local hook you might want to think about moving this into `src/screens/profile_details/hooks.ts`

I will require testing in this file due to these changes